app/src/main/java/com/example/mycalculator/mvp/CalculatorModelImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.example.mycalculator.mvp;
+
+public class CalculatorModelImpl implements CalculatorContract.CalculatorModelContract {
```

Comment:
you can change the name of this class and call it **CalculatorModel**

Reply:
Done. Thx!